src/main/java/Deadlines.java

```diff
@@ -0,0 +1,20 @@
+public class Deadlines extends Task {
```

Comment:
Can consider changing class name to Deadline instead since it only contains 1 deadline